.github/workflows/integrate.yaml

```diff
@@ -75,4 +75,4 @@ jobs:
             dist-newstyle
           key: ${{ runner.os }}-cabal
       - name: Build the full ci derivation
-        run: nix-build ./nix/ci.nix
+        run: nix build -L .#check.x86_64-linux --extra-experimental-features nix-command --extra-experimental-features flakes
```

Comment:
It's probably better to add the following to the `extra-nix-config` variable:
```
experimental-features = nix-command flakes
```

Reply:
See https://github.com/mlabs-haskell/Bondly-NFTBridge/blob/dcf50461b549758f4c9f86fd1a0058db2de89c10/.github/workflows/integrate.yaml#L53